src/components/select/index.md.njk

```diff
@@ -21,6 +21,8 @@ The select component allows users to choose an option from a long list. Before u
 
 Asking questions means you’re less likely to need to use the select component, and can consider using a different solution, such as [radios](../radios/).
 
+In the select component, an option is always selected, even when a user sees it for the first time. This is because its options are settings - unlike the options in radios and [checkboxes](https://design-system.service.gov.uk/components/checkboxes/), which are answers to questions.
```

Comment:
Yeah the copy looks good! 

Just wondering if we needed to flag that idea that an option will be selected "by default", that the user would not have selected?

Reply:
is it true that an option is always selected in selects? The example is a setting, and would always have a value, but I think if someone used it for a question it wouldnt have a default value?

Reply:
Suggesting something like this, based on @36degrees's and @joelanman's feedback:

  ```suggestion
When select is used for settings, an option may be pre-selected when users first see the component.

However, when select is used for questions, none of the options should be pre-selected - in case it influences users' answers.
```

Reply:
```suggestion
If you use the component for settings, you can make an option pre-selected by default when users first see it.

If you use the component for questions, you should not pre-select any of the options in case it influences users' answers.
```

@EoinShaughnessy 
Tried to make this a bit more active voice, what do you think?

Reply:
@calvin-lau-sig7 Way better! Thanks. 👍🏻